xlive/Blam/Engine/memory/static_arrays.h

```diff
@@ -73,18 +73,11 @@ class c_static_stack
 	t_type m_data[k_maximum_count];
 };
 
-
-
 template<typename t_type, typename t_storage_type, size_t k_count>
 class c_flags_no_init 
 {
 	static_assert(std::is_unsigned_v<t_storage_type>, "error: t_storage_type is not an unsigned integral type");
-
-public:
-	const t_storage_type get_unsafe(void) const
```

Comment:
Please leave the get_unsafe and set_unsafe functions in the class

Reply:
It's in the class in the original game and I'd like to keep them in the original class.